tests/posix/basic/basic.cpp

```diff
@@ -135,7 +135,7 @@ TEST_F(basic, open_create_close)
 							{0100777, 1, 0, "bbb"},
 						}));
 
-	EXPECT_TRUE(test_pmemfile_stats_match(pfp, 3, 0, 0, 0));
+	EXPECT_TRUE(test_pmemfile_stats_match(pfp, root_count() + 2, 0, 0, 0));
```

Comment:
I think you should move root_count() to test_pmemfile_stats_match

Reply:
Good idea